frontend/src/components/Popup/Popup.tsx

```diff
@@ -0,0 +1,29 @@
+import "./Popup.scss";
+import { motion } from "framer-motion";
+interface PopupProps {
+  children?: React.ReactNode;
+  onClose: () => void;
+}
+
+export default function Popup(props: PopupProps) {
```

Comment:
Is this something we can insert anywhere?

Reply:
Yeah! The popup component should work on any screen. Just wrap it in an `AnimatePresence` and you'll get the animations too.